Tests/Engine/CustomizedRule.tests.ps1

```diff
@@ -197,8 +201,9 @@ Describe "Test importing correct customized rules" {
             $violations[0].RuleSuppressionID   | Should -Be "MyRuleSuppressionID"
         }
 
-        if (!$testingLibraryUsage)
-		{
+        # if (!$testingLibraryUsage)
+		# {
```

Comment:
Should this still be here?

Reply:
Fixed in b5407379293aa0c3d9b2451a979c82180dfb1e73